libs/remix-simulator/src/utils/logs.ts

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+import gray from 'ansi-gray'
+import timestamp from 'time-stamp'
+import supportsColor from 'color-support'
+
+export class Logger {
```

Comment:
Same here. I'm not sure it makes that much sense to have a class.

Reply:
updated